Libraries/LibGUI/GTextEditor.h

```diff
@@ -244,10 +248,11 @@ class GTextEditor
     RefPtr<GAction> m_delete_action;
     CElapsedTimer m_triple_click_timer;
     NonnullRefPtrVector<GAction> m_custom_context_menu_actions;
-    NonnullOwnPtrVector<NonnullOwnPtrVector<UndoCommand>> m_undo_stack;
-    int m_undo_index = 0;
+    NonnullOwnPtrVector<UndoCommandsContainer> m_undo_stack;
+    int m_undo_stack_size = 0;
```

Comment:
This name is confusing since it collides with `m_undo_stack.size()` which means something different.
Perhaps `m_undo_index` and `m_redo_index` would be better names?